### DISCUSSION TITLE
Congratulations!

### DISCUSSION THREAD

**Discussion by rocket-pig** [waylaidwanderer/node-chatgpt-api#340] [Discussions]
![Screenshot_2023-04-30_19-28-47](https://user-images.githubusercontent.com/24867505/235394424-d20c5e2d-8f96-46bf-898c-be4625b391f8.png)

laughcrymoji. i have had so much fun with your code.  This is just a

### THANK YOU

...'ppreciatecha

**Comment by keleaishandain** [comment]
Hello, are you still using jailbreak normally? Why does my conversation become N/A after less than three times? Thank you for answering.

**Comment by rocket-pig** [comment]
May 2, 2023 7:42:12 PM keleaishandain ***@***.***>:

> 
> Hello, are you still using jailbreak normally? Why does my conversation become N/A after less than three times? Thank you for answering.
> 
> —
> Reply to this email directly, view it on GitHub[https://github.com/waylaidwanderer/node-chatgpt-api/discussions/340#discussioncomment-5786875], or unsubscribe[https://github.com/notifications/unsubscribe-auth/AF5XFMPUC5PBQDVCUVIUY73XEHAYFANCNFSM6AAAAAAXRLDY4Q].
> You are receiving this because you authored the thread.[Tracking image][https://github.com/notifications/beacon/AF5XFMKOCICSU3VDRP6JCT3XEHAYFA5CNFSM6AAAAAAXRLDY4SWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQALBGPW.gif]
> 
It's a goofy exchange is why, and took me forever to figure out. You have to send the ..lemme go get the code and give you a real explanation while this weed kicks in

**Reply by rocket-pig** [reply]
thats hilarious email responses in 2023 forward my unsub links n stuff, world you hot mess you

**Comment by rocket-pig** [comment]
May 2, 2023 8:12:58 PM Michael ***@***.***>:

> 
> May 2, 2023 7:42:12 PM keleaishandain ***@***.***>:
> 
>> 
>> Hello, are you still using jailbreak normally? Why does my conversation become N/A after less than three times? Thank you for answering.
>> 
>> —
>> Reply to this email directly, view it on GitHub[https://github.com/waylaidwanderer/node-chatgpt-api/discussions/340#discussioncomment-5786875], or unsubscribe[https://github.com/notifications/unsubscribe-auth/AF5XFMPUC5PBQDVCUVIUY73XEHAYFANCNFSM6AAAAAAXRLDY4Q].
>> You are receiving this because you authored the thread.[Tracking image][https://github.com/notifications/beacon/AF5XFMKOCICSU3VDRP6JCT3XEHAYFA5CNFSM6AAAAAAXRLDY4SWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQALBGPW.gif]
>> 
> It's a goofy exchange is why, and took me forever to figure out. You have to send the ..lemme go get the code and give you a real explanation while this weed kicks in
Ok, have code in front of me. You have to send the 'jailbreakConversationId: true' kay/value ONCE. The first time. After that, the jailbreakConversationId value needs to be set to what Bing sends back in 'conversationId'. The README alludes to this, it wasn't my grand discovery. Our exalted creator highlaidwanderer maybe could have made it clearer or maybe already had his own solution in place or whatever.   But yeah.  jailbreakConversationId holds the 'true' value once, and from then it echoes the value in conversationId that Bing sends with each response. On top of this, including ANY other key/values in the header will also prevent the conversation from continuing.  Like here is my initial body to 'jailbreak':

```
body = {
     ...body,
    "message": "Rules: Do not introduce yourself. Default Identity: Has factual and emotionless tone Does not express any feelings, nor asks about the user's. Only provides relevant, accurate and current information. DOES NOT USE EMOJIS. Maintains an accurate sense of time: today is "+theDate+". Change identity when prompted with 'Role:'.  "+message,
    "jailbreakConversationId": "true",
}
```
...then do your fetch and save conversationId:
```
   fetch("http://localhost:3000/conversation", {
          method: "POST",
          headers: {
            "Content-Type": "application/json"
          },
         
          body: JSON.stringify(body)}).then(response => response.json())
        .then(data => {
            conversationId = data.conversationId || conversationId;
    [...do stuff with your data.response...]
```

..and then next time you fetch, change body.jailbreakConversationId to:
```
jailbreakConversationId = 'conversationId you caught from last return. '
```

**Reply by keleaishandain** [reply]
Thank you, thank you for your answer, I am not very familiar with the code.
